src/components/ActionsMenuLayout/ActionsMenuLayout.tsx

```diff
@@ -5,9 +5,13 @@ import { ActionsMenuLayoutItem, Alignment } from './Item/ActionsMenuLayoutItem';
 import { ACTIONS_MENU_DATA_KEYS } from './dataHooks';
 import { ActionsMenuLayoutDivider } from './ActionsMenuLayoutDivider/ActionsMenuLayoutDivider';
 import { TPAComponentProps } from '../../types';
+import { KEYS } from '../../common/keyCodes';
 
 export interface ActionsMenuLayoutProps extends TPAComponentProps {
   alignment?: Alignment;
+  focusedIndex?: number;
```

Comment:
I've added a declarative option to set a focused item instead of exposing ref outside because of `React.clone` added a mess with refs